src/core/qgsfieldformatter.h

```diff
@@ -49,6 +80,13 @@ class CORE_EXPORT QgsFieldFormatter
 
     virtual ~QgsFieldFormatter() = default;
 
+    //! Abilities of the formatter
```

Comment:
Can you add `\since QGIS 3.12` to all new flags, classes, structs, public methods?

Reply:
When the class is new and defined as \since QGIS 3.12, do I need them for it's members?

Reply:
no

Reply:
done for the flags...